src/bootstrap.tsx

```diff
@@ -687,6 +688,11 @@ const renderApp = async () => {
       i18n.changeLanguage(defaultLanguage);
     }
 
+    const printAppTemplates
```

Comment:
The name of the variable is misleading, it's a single name of a print app and not a set of templates.